frontend/src/Utilities/Number/minNumberInArray.js

```diff
@@ -0,0 +1,7 @@
+export default function minNumberInArray(numbers) {
+  if (!numbers) {
+    return;
+  }
+
+  return Math.min(...numbers);
```

Comment:
Seems like a waste making an entire new function/class when we can just call `Math.min` in place